src/visualization/plot_recipes.jl

```diff
@@ -362,7 +362,7 @@ end
 
   # Set series properties
   seriestype := :path
-  linecolor := :black
+  linecolor := :grey
```

Comment:
Can you please post 2-3 examples of how this looks like for a 2D plot (before/after comparison)?

Reply:
Sure here are some examples:
First for simple advection:
![alt bl10 30+](https://user-images.githubusercontent.com/72009492/125472914-f1d3961f-a109-4f57-82ef-9830956069e2.PNG)
![amr neu + bl9 40-](https://user-images.githubusercontent.com/72009492/125472928-57407af8-04de-4358-b377-2da22c848a14.PNG)

Reply:
Thanks! I guess it makes sense to use grey as the default when using the default color scheme `inferno`, as the blackish part renders the mesh invisible otherwise.